.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: php
 
 sudo: false
+dist: trusty
```

Comment:
FYI, I have deliberately avoided these (let them go for now) as it will be irrelevant next month anyway, and leaves what will become legacy external configuration in our CB.